contents/blog/startup-finance-without-finance.md

```diff
@@ -0,0 +1,210 @@
+- **Balance sheet**
+  - For an early stage startup, this report basically only tells you how much money you have in the bank. You should not be waiting once a month for your accountant to tell you this!
+  - If you have a foreign subsidiary, you might see an 'intercompany transfer' in there - that is money you've sent from your parent to subsidiary company to cover things like payroll. When it comes to tracking performance vs. your budget, this can be messy (see below!).
+- **Cash flow statement**
+  - This again is not very interesting for an early stage startup (unless you are in fintech/hardware)
+  - The main number that you're interested in here is 'Net cash increase for period' (or similar title) as this is what is also often referred to as 'burn'.
+
+### General tips
+
+- Your accountants are not mind readers - they do not have any context about the things your business is doing, so if there is an unusual transaction they haven't seen before, you'll need to explain to them what it's for. 
+- Be super clear with reporting timeline expectations from your accountant and agree these up front. Similarly, when they need stuff from you, ask them for a deadline - some of these are actually real, legal deadlines!
+- Ask for quarterly and annual reports to be generated, not just monthly. You'll need these for random things like investor data requests throughout the year. 
+- If you are a B2B SaaS company, ask your accountant to put your hosting costs in COGS ('cost of goods sold') not IT expenses. Otherwise it'll look like you have a ridiculous 99% profit margin or something. 
+- If you pay a year up front for an annual subscription, make sure your accountants 'amortize' this over the whole year. This means they'll put 1/12 of the annual cost into each month's expenses. They should also do this if a customer pays an annual contract up front - but you'll need to tell them this. 
+- You may hear 'accruals' vs. 'cash'-based accounting floating around - this basically means you either record the revenue when you issued the invoice vs. when you actually received the cash in the bank. We use accruals-based accounting, which is usually the sensible choice for B2B SaaS. 
+
+## Predicting the future - aka Financial Planning
+
+I lied when I said accounting was the good stuff - _this_ is the good stuff. Let's get our Wall Street on. 
+
+For this section, I'm going to assume you have your accounts set up in QuickBooks. Next, go to your account in Pry - the first thing you'll be prompted to do is connect Pry to your QuickBooks account, which should be straightforward. You can also connect your bank accounts directly if you want to - I like this because it means that Pry always has the latest bank account balances in one place, which is easier than logging into each one and adding up the total. 
```

Comment:
```suggestion
Go to your account in [Pry](https://pry.co/) - the first thing you'll be prompted to do is connect Pry to your QuickBooks account, which should be straightforward. You can also connect your bank accounts directly if you want to - I like this because it means that Pry always has the latest bank account balances in one place, which is easier than logging into each one and adding up the total. 
```